stable/prometheus-redis-exporter/templates/deployment.yaml

```diff
@@ -22,12 +22,6 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ template "prometheus-redis-exporter.serviceAccountName" . }}
-      {{- if .Values.image.pullSecrets }}
```

Comment:
Why did you remove this?

Reply:
Hi CVirus,
i've corrected this. We made some changes yesterday without having the project forked. Today, we just copied the files into our forked version and this has been overriden.

I've added the imagePullSecret-Parts back to the deployment. Thanks for your feedback.